docs/interfaces.md

```diff
@@ -107,6 +107,17 @@ Auto-Connect: no
 
 ## Supported Interfaces - Advanced
 
+### browser
```

Comment:
Regarding the name, yeah, I had a tough time with this one. It doesn't fit control, observe, bind, or manage. I'm definitely open to suggestions.